themes/duke/templates/individual--dukecv-gift.ftl

```diff
@@ -8,34 +8,92 @@
 <#include "individual-setup.ftl">
 <#import "lib-vivo-properties.ftl" as vp>
 
+
 <section id="topcontainer" class="main-content document">
   <section id="individual-info" ${infoClass!} role="region">
     <header>
       <h1><@p.label individual false 1 /></h1>
+      <#-- Description -->
+      <#assign description = propertyGroups.pullProperty("http://vivo.duke.edu/vivo/ontology/duke-cv-extension#description")!> 
```

Comment:
If you look at the other code, there's a lot of long url looking calls like that in the pullProperty method.  It's more like a namespace (like you said "identifier") than a URL.  

I think there is a way to define the namespace as a variable and substitute it in e.g. ${core} or ${namespaces.vitroPublic} - but it doesn't look like the convention is to do that very often.  

I think that's because it can be annoying looking up the true value of ${core} (for instance) when you need to run/debug SPARQL  It's even worse substituting "obo:RO_0000053" type properties.  But the same idea.